files-csharp/CountFilesInaFolderUsingCSharp/CountFilesInaFolderBenchmarks/Program.cs

```diff
@@ -0,0 +1,4 @@
+﻿using BenchmarkDotNet.Running;
+using CountFilesInaFolderBenchmarks;
+
+var summary = BenchmarkRunner.Run<FileCountingBenchmark>();
```

Comment:
No need to store the summary since we are not doing anything with it. We can just call Run directly